Makefile

```diff
@@ -95,6 +95,9 @@ deploy: install-apis ## Deploy the operator to the current cluster
 
 run: build-local-container kind-load deploy ## Build image and run operator in-cluster
```

Comment:
Also add `cert-mgr` to the `run` target?

Reply:
Yes, I'll add it.